### PR TITLE
Speculative fix: Selenium fails to locate expected dropdown auto-suggestions

### PR DESCRIPTION
**Pull request by aweng98**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Suggests.scala [hunk 1]
```diff
@@ -49,10 +48,10 @@ trait Suggests extends LazyLogging { this: Component =>
         " Is this input field enabled for suggestions? Has your test activated the suggestions dropdown?")
     }
 
-    val listOptionXpath = s"#$dropdownId li [data-test-id='$suggestionTestId']"
-    val li = find(cssSelector(listOptionXpath)).head
-
-    logger.info(s"(remove) getSuggestionByTestId: dropdown list contains ${li.size} li: $li")
+    val listOptionXpath = s"""#$dropdownId li *[data-test-id=\"$suggestionTestId\"]"""
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Suggests.scala]: enclose `data-test-id` value in escaped double quotes.

**Comment by aweng98** [comment]
firecloud-ui build failed.

jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala [hunk 2]
```diff
@@ -136,18 +136,15 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
     val statusOption = billingPage.waitForCreateDone(billingProjectName)
 
     statusOption match {
-      case None if trials > 1 =>
-        log.info(s"timeout creating project $billingProjectName, retrying ${trials-1} more times")
-        createNewBillingProject(user, billingPage, trials-1)
-      case Some("failure") if trials > 1 =>
-        log.info(s"failure creating project $billingProjectName, retrying ${trials-1} more times")
+      case None | Some("failure") if trials > 1 =>
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala:139]: unrelated to auto-suggestion. revert my last commit to restore previous code.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Suggests.scala [hunk 3]
```diff
@@ -49,10 +50,13 @@ trait Suggests extends LazyLogging { this: Component =>
         " Is this input field enabled for suggestions? Has your test activated the suggestions dropdown?")
     }
 
-    val listOptionXpath = s"#$dropdownId li [data-test-id='$suggestionTestId']"
-    val li = find(cssSelector(listOptionXpath)).head
+    val listOptionCss1: String = s"""#$dropdownId li *[data-test-id=\"$suggestionTestId\"]"""
+    val listOptionCss2: String = s"""[data-test-id=\"$suggestionTestId\"]"""
+    val alternateCss = Random.shuffle(List(listOptionCss1, listOptionCss2)).head
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Suggests.scala:55]: Selenium method `find` fails to find dropdown list options on rare occasions (speculate reason may be the http url). Providing alternative css locators and use double quotes to work around `find` failure.

**Review by gdlex4015 — APPROVED** [review]

**Pull request merged**